### PR TITLE
Activities ed program info 186900926

### PR DESCRIPTION
**Pull request by guiburato**
#### 🔗 Pivotal Tracker ticket
[#186900926](https://www.pivotaltracker.com/story/show/186900926)

#### ✍️ Description
new activities-ed-program-info screen.

#### 📷 Design reference
[figma](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=748-12962&mode=design&t=woYpAbF4M0LvuCsR-4)

#### ✅ Completion tasks

- [ x ] Added relevant tests
- [ x ] Meets acceptance criteria
- [ x ] Added testing instructions to ticket

### CODE REVIEW

File: src/main/resources/templates/gcc/activities-class-weekly-schedule.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title=#{activities-add-ed-program-type.header})}"></head>
```
Comment (enyia21) [inline src/main/resources/templates/gcc/activities-class-weekly-schedule.html:3]: I don't think that the titles of the pages should be the headers.   `Weekly Class Schedule` may be more appropriate for a page title than 1What days do you take classes?`

File: src/main/resources/templates/gcc/activities-ed-program-info.html [hunk 2]
```diff
@@ -1,18 +1,25 @@
 <!DOCTYPE html>
 <html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<head th:replace="~{fragments/head :: head(title='TEMPLATE')}"></head>
```
Comment (enyia21) [inline src/main/resources/templates/gcc/activities-ed-program-info.html:3]: Same comment as on the [activities-weekly-class-schedule](src/main/resources/templates/gcc/activities-class-weekly-schedule.html)

**Review by enyia21 — DISMISSED** [review]
I think that the title and the headers of pages should be different.   Other than that I think this is good to go.  Although I also wonder if have validations included in the page acceptance testing

File: src/main/resources/templates/gcc/activities-class-weekly-schedule.html (hunk 1 above)
Comment (guiburato) [reply src/main/resources/templates/gcc/activities-class-weekly-schedule.html:3]: Hum... Good call; I'll change the headers based on their names, like your suggestion.

File: src/main/resources/messages.properties [hunk 3]
```diff
@@ -194,6 +194,13 @@ children-ccap-in-care.header=Is {0} already in child care with your chosen provi
 #activities-ed-program-type
 #activities-ed-program-name
 #activities-ed-program-info
+activities-ed-program-info.header=Program information
+activities-ed-program-info.description=You can skip questions that are not relevant to your program.
+activities-ed-program-info.phone-number=What is their phone number?
```
Comment (sree-cfa) [inline src/main/resources/messages.properties:199]: Copy is slightly different in the figma
```suggestion
activities-ed-program-info.phone-number=What's their phone number?
```

File: src/main/java/org/ilgcc/app/inputs/Gcc.java [hunk 4]
```diff
@@ -7,24 +7,23 @@
...
     @NotBlank(message = "{activities-ed-program-method.validationMessage}")
     private String programSchedule;
-
     private String languageRead;
     private String languageSpeak;
+    private String phoneNumber;
+    private String streetAddress;
+    private String city;
+    private String state;
+    private String zipCode;
```
Comment (sree-cfa) [inline src/main/java/org/ilgcc/app/inputs/Gcc.java:26]: There's a comment in figma about validating this field, we can use the error message from here --> [error message library](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=792-45252&mode=design&t=qQzT01wP9z0WzljC-0)

File: src/main/resources/messages.properties (hunk 3 above)
Comment (guiburato) [reply src/main/resources/messages.properties:199]: I follow the pattern of other fields. Should I follow exactly what the figma looks like?

Comment (sree-cfa) [reply src/main/resources/messages.properties:199]: Yeah generally - when Carlie gets back, we can ask her if that should be changed

File: src/main/resources/messages_es.properties [hunk 5]
```diff
@@ -169,7 +177,9 @@ activities-ed-program-method.validationMessage=Por favor selecciona una opcion
 activities-next-class-schedule.header=A continuaci�n, le preguntaremos por su horario de clases.
 activities-next-class-schedule.description=<b>Dado que su horario de clases es flexible</b>, seleccione los d�as y horas en que planea asistir a clases o estudiar mientras su hijo est� bajo cuidado. 
 activities-next-class-schedule.skip=Omitir porque no necesito cobertura de cuidado de ni�os durante el horario de clases.
+
 #activities-class-weekly-schedule
+activities-class-weekly-schedule.title=Horario de clases semanal
```
Comment (sree-cfa) [inline src/main/resources/messages_es.properties:182]: Let's leave this out for now until we get a translation from cesar - there's some context around language accessibility that we want reviewed by someone in research

File: src/main/resources/templates/gcc/activities-ed-program-info.html [hunk 6]
```diff
@@ -1,18 +1,25 @@
...
+        <th:block th:replace="~{fragments/inputs/headerNameUnderlined :: headerNameUnderlined(text=${fieldData.schoolName})}"></th:block>
+        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{activities-ed-program-info.header}, subtext=#{activities-ed-program-info.description})}"/>
         <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
-                <!-- Put inputs here -->
+              <th:block th:replace="~{fragments/inputs/phone :: phone(inputName='phoneNumber', label=#{activities-ed-program-info.phone-number})}"/>
+              <th:block th:replace="~{fragments/inputs/text :: text(inputName='streetAddress', label=#{activities-ed-program-info.street-address})}" />
+              <th:block th:replace="~{fragments/inputs/text :: text(inputName='city', label=#{activities-ed-program-info.city})}" />
+              <th:block th:replace="~{fragments/inputs/state ::
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-ed-program-info.html:19]: Looks like we need to overwrite the placeholder here in message properties 
`state.placeholder=Select the state`

File: src/main/resources/messages_es.properties (hunk 5 above)
Comment (guiburato) [reply src/main/resources/messages_es.properties:182]: Ok

File: src/main/resources/templates/gcc/activities-ed-program-info.html (hunk 6 above)
Comment (guiburato) [reply src/main/resources/templates/gcc/activities-ed-program-info.html:19]: ok, Done.

**Review by sree-cfa — APPROVED** [review]
lgtm

**Pull request merged**